### PR TITLE
Updates for theme

### PR DESCRIPTION
**Pull request by Grizzly-Adam**
Added three more options for themes: taskbar_clock_position_and_size, exit_character, and titlebar_textcolor. Added four themes based on various operating systems.

### CODE REVIEW

File: apps/launcher_app.lua [hunk 1]
```diff
@@ -55,12 +55,12 @@ laptop.register_app("launcher", {
 		end
 		if app.app_info then
 			if #mtos.sysram.stack > 0 then
-				formspec = formspec.."label[0.8,-0.29;"..app.app_info.."]"
+				formspec = formspec..mtos.theme:get_label("0.8,-0.29", app.app_info, "titlebar")
 			else
-				formspec = formspec.."label[-0.1,-0.29;"..app.app_info.."]"
+				formspec = formspec..mtos.theme:get_label("0.8,-0.29", app.app_info, "titlebar")
```
Comment (bell07) [inline apps/launcher_app.lua]: The old position value was "-0.1,-0.29". Te titlebar text is more at the left site if no back button visible

**Comment by bell07** [comment]
I tested the new skins, F*cking nostalgia, I did not seen Win-98 since ~15 years. Really Great Work!
Please adjust the label position if no "Back" button there as mentionend in review, and please add the new theming parameter to the  [API.md](https://github.com/Gerold55/minetest-laptop/blob/master/API.md), then the merge is approved from my site

File: API.md [hunk 2]
```diff
@@ -113,6 +113,10 @@ Definitiontable:
 - `major_button` Major (highlighted) button image
 - `minor_button` Minor button image
 - `node_color` Palette number to set if the node is paramtype2 = "colorfacedir"
+- `taskbar_clock_position`
```
Comment (bell07) [inline API.md]: taskbar_clock_position_and_size 

Pls. add short description what the setting does

**Comment by Grizzly-Adam** [comment]
I will. Working on it during breqks at work.

**Comment by Grizzly-Adam** [comment]
Ok, it is ready.

**Comment by bell07** [comment]
The attribute is "taskbar_clock_position_and_size", not "taskbar_clock_position"

**Comment by Grizzly-Adam** [comment]
Copy and paste fail. It should be good to go now.

**Comment by Grizzly-Adam** [comment]
Anything else I need to do or will all this fall into place now?

**Pull request merged**